### ISSUE TITLE
Automatically skip file if already exists

### ISSUE THREAD

**Issue by git-reader** [erdnuesse/flac-to-mp3#1]
Suggest the following file test in front of the encoding command:
`## RUN`

`[ -f "$outfile" ] || ffmpeg -loglevel info -i "$fullfile" -codec:a libmp3lame -qscale:a 0 "$outfile"`

Will now automatically skip existing files without user intervention so the script can be run daily, monitoring the original location for additional files.
Thanks for taking the time to publish your script; it's helped me! :+1:

**Comment by erdnuesse** [comment]
Hi, and thanks for the comment. The purpose for my Initial scripts was a one-time conversion. I did not think about a service or cronjob. I will have a look when I find the time, but it looks good so far.
Do you want to commit that change yourself? I am not exactly familiar with github I have to admit. 🤷🏽‍♂️😬

**Comment by erdnuesse** [comment]
Ok, now i DID take a look into it, and it seems that you did not have to do that. 
As seen [here](https://ffmpeg.org/ffmpeg.html#Main-options), when you use ffmpeg, -n takes care of your if statement and might therefore be faster than your solution. I included -n also in my final run command, since the options in avconf and ffmpeg are mostly the same, even though there is no documentation on the -n switch to be found on the internet, yet it still worked. 

Command:
`avconv -n -nostats -loglevel info -i "$fullfile" -codec:a libmp3lame -qscale:a 0 "$outfile"`

First I was confused, because, whenever I tested my script, I already skipped existing files instead of overwriting them.

Issue therefore closed, 
TL;DR, do not use this suggestion, it's been taken care of already in the original script. 

Regards, Kay
